### ISSUE TITLE
Not using DbConnection for the connector and therefor Dapper is not supported...

### ISSUE THREAD

**Issue by MrBananaGuard** [SapientGuardian/mysql-dnxcore50#4]
Hi,

I was hoping yours was just a decompiled version of the one that oracle has provided, but a lot of the base classes from the official version are not there. Such as on the MySqlConnection class yours inherits: RTConnection which I am assuming is a class you wrote, but it needs to inherit: DbConnection.

I am going to do a pull of your version and try and make it use the same types as the official version.

**Comment by SapientGuardian** [comment]
It isn't decompiled, it's a fork of the official sources. I wrote no new code for this, but the ones that are present may not be the ones you're used to from the full desktop version. .NET Core is closer to WinRT than it is to the full framework, so I ended up using a lot of the RT implementations. The version I'm working on in https://github.com/SapientGuardian/mysql-connector-net-netstandard1.5 is less butchered than this one, but it probably still uses RTConnection.
